gdk4/src/texture.rs

```diff
@@ -5,6 +5,9 @@ use crate::Texture;
 use glib::translate::*;
 use glib::IsA;
 
+unsafe impl Send for Texture {}
+unsafe impl Sync for Texture {}
```

Comment:
Do something like this instead:

https://github.com/gtk-rs/gtk-rs-core/blob/c53f692d84f7f7a3a75362cd424427842f792e66/gio/Gir.toml#L331

Also the same for all `GdkTexture` subclasses (memory, gl)

Reply:
Done, thanks.